### DISCUSSION TITLE
Unauthorized error on second submition attempt

### DISCUSSION THREAD

**Discussion by rross128** [livewire/livewire#3898] [1. Help]
I have a simple "Create and New" Livewire component, the kind where it saves the info, clears out the form so you can keep adding more without having to close and refresh. The form has an authorization and validation method for the obvious reasons.

On the first time through, the form completes the authorization and validation and successfully creates the new object. The second time through, the form blows up with a 403 authorization error.

Anyone have any thoughts?

**Comment by joshhanley** [comment]
@rross128 please share your component class and blade view.

Thanks!

**Comment by rross128** [comment]
Do you know where you're reviewing something really carefully before you post it just to make sure you're not doing something completely idiotic you might get called out on? 

Well, that is what I was doing when I happened to notice that the object I was using to authorize the request on had ended up in the function to clear the user input, thus being set to an empty string, thus meaning there was no real Model to perform an authorization request on for the second request, thus my error. /sigh

Thanks for taking the time to look, @joshhanley. Appreciate it.

**Reply by joshhanley** [reply]
@rross128 no worries 🙂
